example/index.html

```diff
@@ -156,6 +163,49 @@
 
     function onAuthData(data) {
         console.log("auth data received: "+data)
+
+        // extract expiry.
+        let expiry = window[namespace].wasmClientGetExpiry();
+        if (!expiry) {
+            document.getElementById('expiry').innerHTML = "No expiry found in macaroon";
+            return {}
+        }
+
+        localStorage.setItem(namespace+":expiry", expiry)
+        document.getElementById('expiry').innerHTML = "Session Expiry: "+new Date(expiry*1000).toLocaleString();
+
+        // extract permissions.
+        let res = window[namespace].wasmClientHasPerms("lnrpc.Lightning.GetInfo")
+        console.log(res)
+        if (window[namespace].wasmClientHasPerms("lnrpc.Lightning.GetInfo")) {
+            document.getElementById('getinfo').disabled = false;
+        }
+        if (window[namespace].wasmClientHasPerms("lnrpc.Lightning.ListChannels")) {
```

Comment:
Great idea with disabling the buttons!
Note to self: Maybe we should import jQuery into this page to make this JS code a bit more pretty :sweat_smile:

Reply:
yeah, agreed that it is getting a bit hairy. But is it ok to leave as is for now since it is just an example?